app/src/index.css

```
@@ -307,14 +314,30 @@
+     Tailwind v3 those utilities compiled into `transform`, so a keyframe
+     setting `transform` *replaced* the centering for the animation's duration
+     and only had to carry the -50% pair itself. Under v4 they compile to
+     `translate: var(--tw-translate-x) var(--tw-translate-y)` instead — a
+     separate property that composes with `transform` rather than being
+     overridden by it. A `transform: translate(-50%, …)` keyframe therefore
+     *added* to the utility, so every modal opened at -100%/-100% (up and to
+     the left of center) and snapped into place when the animation ended.
+
+     Animating `translate` overrides the same property the utility sets and
+     lands on the identical value, so there is nothing left to snap. */
   @keyframes dialogIn {
```

Stylelint (17.14.0)

[error] Expected keyframe name "dialogIn" to be kebab-case (keyframes-name-pattern)

(keyframes-name-pattern)